compiler/prelude/genmin.go

```diff
@@ -0,0 +1,63 @@
+// +build ignore
+
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"go/build"
+	"io/ioutil"
+	"log"
+	"os/exec"
+	"path/filepath"
+	"strings"
+
+	"github.com/gopherjs/gopherjs/compiler/prelude"
+)
+
+func main() {
+	if err := run(); err != nil {
+		log.Fatalln(err)
+	}
+}
+
+func run() error {
+	bpkg, err := build.Import("github.com/gopherjs/gopherjs", "", build.FindOnly)
+	if err != nil {
+		return fmt.Errorf("failed to locate path for github.com/gopherjs/gopherjs: %v", err)
+	}
+
+	preludeDir := filepath.Join(bpkg.Dir, "compiler", "prelude")
+
+	args := []string{
+		filepath.Join(bpkg.Dir, "node_modules", ".bin", "uglifyjs"),
+		"--config-file",
+		filepath.Join(preludeDir, "uglifyjs_options.json"),
+	}
+
+	stderr := new(bytes.Buffer)
+	cmd := exec.Command(args[0], args[1:]...)
+	cmd.Stdin = strings.NewReader(prelude.Prelude)
+	cmd.Stderr = stderr
+
+	byts, err := cmd.Output()
```

Comment:
This isn't a big deal, but it's pretty unusual to see a `[]byte` variable named `byts`. As far as I understand, it's a shortened version of "bytes"? It comes up a total of 5 times in the Go project.

In this context, just `b` or `out` would be more idiomatic and clear I think.

I see that `out` is already taken below, in that case, I would make this variable name more specific. For example, `minified`.

The reason I point this out is because I think it's a good idea to strive to write Go code that looks like Go code other Go programmers are most likely to write.

Reply:
Fixed.